src/components/ContextBar.js

```diff
@@ -19,50 +25,77 @@ const contextBarStyles = {
   }
 }
 
-export const ContextBarButtons = ({ setDeletingWorkspace, setCloningWorkspace, setSharingWorkspace, isOwner, canShare }) => {
```

Comment:
not sure why git decided to not identify the diffs when all I really did was rename this component.... Ill call out the actual different stuff inline below